Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrk_Output_cff.py

```diff
@@ -11,14 +11,17 @@
     outputCommands = cms.untracked.vstring( 
         'keep *_IsoProd_*_*',
         'keep *_TkAlIsoProd_*_*',
+        'keep *_hbhereco_*_*',
```

Comment:
The output from previous output command was not useful for calibration purpose. This will increase the o/p but the trigger rate is small enough